docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`-` Rotki will now support more Yearn vaults due to improved decoding logic.
```

Comment:
Don't add this. You already added yearn v3 earlier. If anything augment that entry.